src/app/(views)/managed-courses/[courseId]/modules/page.tsx

```diff
@@ -16,7 +39,8 @@ export default function Modules() {
   }
 
   return (
-    <table>
+    <>
+      {/* <table>
```

Comment:
Do we need this commented code? if yes, put a comment above explaining why this is still here.